particles/Tutorial/Javascript/3_RenderSlots/RenderSlots.arcs

```diff
@@ -6,18 +6,18 @@ particle ParentParticle in 'parent.js'
   // This particle renders to the root slot ("consumes" it), and provides a slot inside it called "mySlot" in which another particle can render
   // itself. The child particle will be rendered inside a special div with the identifier "mySlot", which this particle will need to provide in
   // its HTML.
-  consume root
-    provide mySlot
+  root: consumes Slot
+    mySlot: provides
 
 // The "child" particle. Instead of consuming "root" it consumes "mySlot", which connects it to the slot provided by ParentParticle.
 particle ChildParticle in 'child.js'
-  consume mySlot
+  mySlot: consumes
 
 recipe RenderSlotsRecipe
   ParentParticle
-    consume root
-      provide mySlot as slot0
+    root: consumes
+      mySlot: provides theSlot
   ChildParticle
-    consume mySlot as slot0
+    mySlot: consumes theSlot
```

Comment:
Why ParentParticle names its main slot `root`, but ChildParticle names its main slot `mySlot`  ? What should be a canonical name for a slot you render your UI to? `surface` ? `ui` ? maybe even `root` ?

Reply:
I don't want to block, but it's good to ask these questions. What do you think @SHeimlich ?

Reply:
I think "root" is confusing since it is the special slot the system provides. I went with render as it seems to be a nice middle ground.